### PR TITLE
[DOCS] Grouped navigation menu - NS

### PR DESCRIPTION
**Pull request by nastasha-solomon**
Addresses my portion of updates for the Grouped Navigation meta (https://github.com/elastic/security-docs/issues/2237).

Previews:
- [Open and manage cases](https://security-docs_2263.docs-preview.app.elstc.co/guide/en/security/master/cases-open-manage.html)
- [Enable threat intelligence integrations](https://security-docs_2263.docs-preview.app.elstc.co/guide/en/security/master/es-threat-intel-integrations.html)
- [About Timeline templates](https://security-docs_2263.docs-preview.app.elstc.co/guide/en/security/master/timeline-templates-ui.html)
- [Migrate detection alerts enriched with threat intelligence](https://security-docs_2263.docs-preview.app.elstc.co/guide/en/security/master/post-upgrade-req-cti-alerts.html)

### CODE REVIEW

File: docs/getting-started/threat-intel-integrations.asciidoc [hunk 1]
```diff
@@ -63,6 +63,6 @@ NOTE: For more information about enabling available threat intelligence filesets
 +
 NOTE: Threat intelligence indices aren’t required to be ECS compatible. However, we strongly recommend compatibility if you’d like your alerts to be enriched with relevant threat indicator information. You can find a list of ECS-compliant threat intelligence fields at {ecs-ref}/ecs-threat.html[Threat Fields].
 
-. Return to the Threat Intelligence view on the Overview page. Refresh the page if indicator data isn't displaying.
+. Return to the Threat Intelligence view on the Overview page (*Dashboards -> Overview*). Refresh the page if indicator data isn't displaying.
```
Comment (joepeeples) [inline docs/getting-started/threat-intel-integrations.asciidoc:66]:
```suggestion
. Return to the Threat Intelligence view on the Overview dashboard (*Dashboards -> Overview*). Refresh the page if indicator data isn't displaying.
```

**Review by joepeeples — COMMENTED** [review]
One small suggested edit, LGTM!

File: docs/getting-started/threat-intel-integrations.asciidoc (hunk 1 above)
Comment (nastasha-solomon) [reply docs/getting-started/threat-intel-integrations.asciidoc:66]: Ah, I was wondering about this. Thanks, will incorporate it now.

**Comment by karanbirsingh-qasource** [comment]
Hi @nastasha-solomon 

we have reviewed the shared preview links and below are observations for them

first 2 links of previews seems to be incorrect as they are pointing to master not our pr, but we have changed them by adding pr numbers in url and then correct doc changes reflected
>Previews:
>- [Open and manage cases](https://www.elastic.co/guide/en/security/master/cases-open-manage.html)
>- [Enable threat intelligence integrations](https://www.elastic.co/guide/en/security/master/es-threat-intel-integrations.html)


**Screen-Shoots:**

image
----
image
-----
image
----
image

----
image


Hence we are adding "QA:Validated" tag to it.

thanks !!

**Comment by nastasha-solomon** [comment]
Thanks for checking this, @karanbirsingh-qasource ! I fixed the first two links so they show the preview now.

**Review by benironside — APPROVED** [review]

File: docs/post-upgrade/post-upgrade-req-cti-alerts.asciidoc [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 == Migrate detection alerts enriched with threat intelligence
 
-After upgrading to {stack} version 7.15.x from a release between 7.12.0 and 7.14.2, you need to migrate detection alerts enriched with threat intelligence data to ensure threat intelligence properly displays in {elastic-sec}.
+After upgrading to {stack} version 7.15.x or higher from a release between 7.12.0 and 7.14.2, you need to migrate detection alerts enriched with threat intelligence data to ensure threat intelligence properly displays in {elastic-sec}.
```
Comment (jmikell821) [inline docs/post-upgrade/post-upgrade-req-cti-alerts.asciidoc:5]: I think `newer` is the preferred term. 

```suggestion
After upgrading to {stack} version 7.15.x or newer from a release between 7.12.0 and 7.14.2, you need to migrate detection alerts enriched with threat intelligence data to ensure threat intelligence properly displays in {elastic-sec}.
```

**Review by jmikell821 — COMMENTED** [review]
One slight nit, then good to merge. Thanks!

**Pull request merged**